File: 01_Easy/0001_Two_Sum.py
class Solution:
    def twoSum(self, nums: list[int], target: int) -> list[int]:
        length = len(nums)
        start_idx = 0
        for i in range(length):
            num = nums[i]
            leftover = target - num
            for j in range(i+1, length):
                if nums[j] == leftover:
                    return [i, j]

File: 01_Easy/0001_Two_Sum.cpp
using namespace std;
#include <vector>

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int curr_num;
        int search_num;
        vector<int> output = {-1, -1};

        for (int i=0; i < nums.size(); i++){
            curr_num = nums.at(i);
            for (int j = i+1; j < nums.size(); j++){
                search_num = target - curr_num;
                if (nums.at(j) == search_num){
                    output = {i, j};
                }
            }
        }
        return output;
    }
};